app/assets/stylesheets/application.css.scss

```diff
@@ -20,6 +20,20 @@
 
 $margin:10%;
 
+a.button.tiny.sign-out {
+  color: $black;
+}
+
+.logo {
+  max-width: 75px;
+  position: relative;
+  top: 5px;
+}
+
+li {
+  margin: 0 0 -8px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 -8px` instead of `0 0 -8px 0`